### ISSUE TITLE
Error when bulk editing issues

### ISSUE THREAD

**Issue by serberman** [apsmir/custom_field_sql#9]
format Sql -> sql request
`select subject from issues where id=%id%`

in production.log
```
ctionView::Template::Error (undefined method `id' for #<Array:0x000001254dd24a10>):
    196:    <% custom_fields.each do |custom_field| %>
    197:      <p>
    198:        <label><%= custom_field.name %></label>
    199:        <%= custom_field_tag_for_bulk_edit('issue', custom_field, @issues, @issue_params[:custom_field_values][custom_field.id.to_s]) %>
    200:      </p>
    201:      <% if i == split_on %>
    202:        </div><div class="splitcontentright">
```

**Comment by NigmatullinAN** [comment]
Sad mistake. Does anyone have a solution?

**Comment by apsmir** [comment]
fixed in version 2.6
